test/lint/commit-script-check.sh

```diff
@@ -17,6 +17,40 @@ if test "x$1" = "x"; then
     exit 1
 fi
 
+function verify_script () {
+  local SCRIPT=$1
+  # use file to trigger error since pipe commands are executed in subshell (can't use vars)
+  local FLAG_FILE=/tmp/CSC_ERROR_$$
+  (
+  export PATH='/invalid'
+  # ignore commands other than 'sed'
+  function command_not_found_handle () {
+    return 0;
+  }
```

Comment:
I don't see any examples where it'd be a problem, but that won't capture any builtin commands.

This will block processing of `git ls | xargs sed` so those cases won't be checked.